test/Winton.Extensions.Configuration.Consul.Test/Extensions/KVPairQueryResultExtensionsTests.cs

```diff
@@ -181,8 +186,9 @@ private void ShouldBeEmptyIfResponseIsNull()
                 _parser
                     .Setup(p => p.Parse(It.IsAny<Stream>()))
                     .Returns(new Dictionary<string, string> { { "key", "value" } });
+                _source.KeyToRemove = "test/path";
 
-                var config = result.ToConfigDictionary("test/path", _parser.Object);
+                var config = result.ToConfigDictionary(_source.ConvertToConfig);
```

Comment:
Here for example it would be sufficient to just do:

```csharp
var config = result.ToConfigDictionary(_ => new Dictionary { {"key", "value"} });
```